extensions/src/org/pentaho/platform/web/http/filters/PentahoPathDecodingFilter.java

```diff
@@ -40,8 +75,39 @@ public void destroy() {
   }
 
   private static final class EncodingAwareHttpServletRequestWrapper extends HttpServletRequestWrapper {
+
+    StringKeyStringValueIgnoreCaseMultivaluedMap parameterMap = null;
```

Comment:
@mbatchelor For every new request we have a new rapper with new map inside. What the problem with threadsafe?